music21/musicxml/m21ToXml.py

```diff
@@ -2469,6 +2471,8 @@ def __init__(self, partObj: Union[stream.Part, stream.Score, None] = None, paren
             self.midiChannelList = parent.midiChannelList  # shared list
             self.makeNotation = parent.makeNotation
 
+        self.previous_sibling_in_group: Optional[stream.PartStaff] = None
```

Comment:
since this is a public interface it should be camelCased.

Reply:
Explaining what a sibling is would be helpful.

Reply:
Yeah, and it's not really any more keystrokes to call it `previousPartStaffInGroup` either...